### PR TITLE
Mivhanei tmicha pipeline

### PR DESCRIPTION
**Pull request by ilyalyudevig**
Hi, 
I made a flow and added it to a pipeline spec, should I run dpp budget key plugin on my local machine to check publication Id generation or it's okay to check it right here?
And what info should be in "Documents" field?

### CODE REVIEW

File: datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/m_tmicha.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#from datapackage_pipelines_budgetkey.common.publication_id import calculate_publication_id
+
+
+def m_tmicha_scraper():
+
+    url = 'https://www.health.gov.il/Subjects/Finance/Mtmicha/Pages/default.aspx'
+    s = requests.Session()
+    page = pq(s.get(url).text)
+
+    for row in page.find('.ms-rtestate-field a'):
```
Comment (akariv) [inline datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/m_tmicha.py]: It's probably better to use an exact id - I think that this class is probably too broad:
(e.g. consider id `ctl00_PlaceHolderMain_GovXContentSectionPanel_ctl00__ControlWrapper_RichHtmlField`)

File: datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/m_tmicha.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        if row.attrib['href'][-3:] == 'pdf':
+            link = 'https://www.health.gov.il' + row.attrib['href']
+            title = row.text
+            yield dict(
+                link=link,
+                page_title=title,
+                publication_id=0,
+                documents='',
```
Comment (akariv) [inline datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/m_tmicha.py]: Please add the documents section with the link to the document.

File: datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/m_tmicha.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    page = pq(s.get(url).text)
+
+    for row in page.find('.ms-rtestate-field a'):
+        if row.attrib['href'][-3:] == 'pdf':
+            link = 'https://www.health.gov.il' + row.attrib['href']
+            title = row.text
+            yield dict(
+                link=link,
+                page_title=title,
+                publication_id=0,
```
Comment (akariv) [inline datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/m_tmicha.py]: add publication id calculation (based on title)

File: datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/m_tmicha.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    url = 'https://www.health.gov.il/Subjects/Finance/Mtmicha/Pages/default.aspx'
+    s = requests.Session()
+    page = pq(s.get(url).text)
+
+    for row in page.find('.ms-rtestate-field a'):
+        if row.attrib['href'][-3:] == 'pdf':
+            link = 'https://www.health.gov.il' + row.attrib['href']
+            title = row.text
+            yield dict(
+                link=link,
```
Comment (akariv) [inline datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/m_tmicha.py]: this should be renamed to `page_url` and point to the web page containing the link (`url`) and not the url of the docuemnt.

File: datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/pipeline-spec.yaml [hunk 5]
```diff
@@ -1,3 +1,10 @@
+m_tmicha_scraper:
```
Comment (akariv) [inline datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/pipeline-spec.yaml]: 1. let's have a more meaningful name than `m_tmicha`
2. Please also incorporate this to the main calls_for_bids pipelines so it gets in the db alongside with the others.

File: datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/m_tmicha.py [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+                page_title=title,
+                publication_id=0,
+                documents='',
+                tender_type='call_for_bids',
+                tender_id=None
+            )
```
Comment (akariv) [inline datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/m_tmicha.py]: missing fields:
- publisher
- tender_type_he
- (maybe more? need to check)

File: datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/m_tmicha.py [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    for row in page.find('.ms-rtestate-field a'):
+        if row.attrib['href'][-3:] == 'pdf':
+            link = 'https://www.health.gov.il' + row.attrib['href']
+            title = row.text
+            yield dict(
+                link=link,
+                page_title=title,
+                publication_id=0,
+                documents='',
+                tender_type='call_for_bids',
```
Comment (akariv) [inline datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/m_tmicha.py]: Note that this should be `support_criteria`, and not `call_for_bids` (which are קולות קוראים - a different concept)

File: datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/m_tmicha.py (hunk 3 above)
Comment (ilyalyudevig) [reply datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/m_tmicha.py]: @akariv sorry for the noise, but seems I didn't get it right - this should be some string generated from title, right?

**Pull request merged**